client/src/modals/signup.jsx

```diff
@@ -0,0 +1,198 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Modal from 'react-modal';
+import axios from 'axios';
+
+import { Button, Tile, Form } from '../styles/globalStyles';
+
+Modal.setAppElement(document.getElementById('app'));
+Modal.defaultStyles.overlay.backgroundColor = 'rgba(0,0,0,0.6)';
+
+export default function Signup() {
+  const [showModal, toggleModal] = useState(false);
+
+  function submitSignup(e) {
+    e.preventDefault();
+    const { email, password, bandname } = e.target.form;
+    const checkboxes = e.target.form.querySelectorAll('input[type=checkbox]:checked');
+    const genres = [];
+    for (let i = 0; i < checkboxes.length; i += 1) {
+      genres.push(checkboxes[i].value);
+    }
+    const data = {
+      name: bandname.value,
+      email: email.value,
+      password: password.value,
+      description: '',
+      genre: genres[0],
```

Comment:
Does this only send the first genre in the array? Should we redesign how the signup endpoint works to handle this?